src/Factories/JsonErrorFactory.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace WtfPhp\JsonApiErrors\Factories;
+
+use Throwable;
+use WtfPhp\JsonApiErrors\Exceptions\JsonErrorException;
+use WtfPhp\JsonApiErrors\Models\JsonError;
+
+/**
+ * Class JsonErrorFactory
+ * @package WtfPhp\JsonApiErrors\Factories
+ */
+class JsonErrorFactory implements JsonErrorFactoryInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public static function createFromThrowable(Throwable $throwable, int $status = 500): JsonError
+    {
+        $jsonError = new JsonError();
+        $jsonError->code = $throwable->getCode();
+        $jsonError->title = $throwable->getMessage();
+        $jsonError->detail = $throwable->getTraceAsString();
+        $jsonError->links = [
+            'about' => sprintf('%s at lint %s', $throwable->getFile(), $throwable->getLine())
```

Comment:
We're not so happy about the link-section. It's supposed to be an actual link, the information about file and line should not go here.

Reply:
We already have the information through the trace, isn't that enough?

Reply:
I moved the about link to the JsonErrorException.